astro/src/content/docs/customize/look-and-feel/index.mdx

```diff
@@ -3,6 +3,7 @@ title: Themes
 description: Learn how to theme and the FusionAuth login pages (including forgot password, two-factor authentication and others)
 section: customize
 subcategory: look and feel
+topOfNav: true
```

Comment:
Same comment about this not really being an overview\index page.  It is the Theme page.